src/ebay-fake-menu/menu-item.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ComponentProps, FC, MutableRefObject, useEffect } from 'react'
+import classNames from 'classnames'
+import { EbayIcon } from '../index'
+
+export type EbayFakeMenuItemProps = Omit<ComponentProps<'a'>, 'onKeyDown'> & {
+    current?: boolean;
+    disabled?: boolean;
+    itemRef?: MutableRefObject<HTMLAnchorElement>;
```

Comment:
It's working but I never seen this pattern before. Is it better if we use boolean props like `autoFocusFirstItem` and do autofocus on first menu item if the prop is true?

- In this way, we don't need to pass a ref to the component. 
- Ref is used internally to focus the first item

Since the value from the given ref is not used by the component (in fact it's ignored and will be replaced with the ref to anchor element), it's cleaner to do it that way.

Reply:
you're right, replaced with `autofocus` prop